region/tourian/main/Metroid Room 1.json

```diff
@@ -643,134 +694,114 @@
+            }}
+          ]},
+          {"and": [
+            {"metroidFrames": 300},
+            {"shinespark": {
+              "frames": 54
+            }}
+          ]},
+          {"and": [
+            "canTrickyJump",
+            {"shinespark": {
+              "frames": 75,
+              "excessFrames": 16
+            }}
+          ]}
+        ]}
       ]
     },
     {
-      "link": [2, 4],
-      "name": "Blue SpaceJump, Into the Room",
-      "entranceCondition": {
-        "comeInShinecharging": {
-          "length": 1,
-          "openEnd": 1
-        }
-      },
+      "link": [3, 2],
+      "name": "Metroid Room 1 Spring Ball Jump (Left to Right)",
       "requires": [
-        "canBlueSpaceJump",
-        "canCarefulJump"
+        "h_canUseSpringBall",
+        "canLateralMidAirMorph",
+        "canCarefulJump",
+        {"or": [
+          {"obstaclesCleared": ["A"]},
+          "canMetroidAvoid",
+          "ScrewAttack",
+          {"metroidFrames": 200}
+        ]}
       ],
-      "note": "Jump Immediately upon entry, or before the transition.",
-      "devNote": "There is 1 unusable tile in this runway."
-    },
-    {
-      "link": [2, 4],
-      "name": "Tank the Damage",
-      "requires": [
-        {"metroidFrames": 421}
+      "note": [
+        "At max run speed without SpeedBooster, you can bounce on both platforms and avoid all acid.",
+        "Jump from the center of the rightmost X in the background.",
+        "Bounce again on the second platform to avoid any Metroid or Rinkas."
       ],
-      "note": "Taking a rinka hit stops the Metroid damage for a while and is less damage."
-    },
-    {
-      "link": [3, 2],
-      "name": "Base",
-      "requires": [
-        "SpaceJump"
-      ]
-    },
-    {
-      "link": [3, 2],
-      "name": "Speedjump",
-      "requires": [
-        "SpeedBooster"
-      ]
+      "devNote": "Just doing the strat avoids almost all damage, so Screw attack and metroidframes values are mostly artificial."
     },
     {
       "link": [3, 2],
-      "name": "Metroid Room 1 Spring Ball Jump (Left to Right)",
+      "name": "Frozen Rinka (Left to Right)",
       "requires": [
-        "canSpringBallBounce"
+        "canTrickyUseFrozenEnemies",
+        "canInsaneJump"
       ],
-      "note": "At max non-speed run speed, you can bounce on both platforms and avoid all acid."
+      "note": [
+        "Doable using the far left Rinka to cross, at a high angle.",
+        "Alternatively, it can also just be used to setup an angle with the Rinka near the left edge of the pool."
+      ]
     },
     {
       "link": [3, 2],
-      "name": "Metroid Room 1 Frozen Rinka (Left to Right)",
+      "name": "Frozen Metroid (Left to Right)",
       "requires": [
-        "canUseFrozenEnemies"
+        {"not": "f_KilledMetroidRoom1"},
+        {"obstaclesNotCleared": ["A"]},
+        "canRiskPermanentLossOfAccess",
+        "canTrickyUseFrozenEnemies",
+        "canTrickyJump"
```

Comment:
This strat is much easier. I would recommend combining them as `Frozen Metroid or Rinka` and then they can share a note and it will tell them to use the Metroid if they're alive.